### ISSUE TITLE
The skill details radio button on the client-display lacks necessary context for screen readers

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1176]
Currently the skill details radio button in the client-display provides no context for screen readers (simply read as "checkbox not-checked/checkbox checked). Either an appropriately attached form label needs to be used for the radio button label (linked via for=id attribute) or the radio button checkbox input needs to include aria-label describing what the checkbox *is*

**Comment by rmmayo** [comment]
PR: #1339
